src/main/java/Duke.java

```diff
@@ -7,12 +7,18 @@
 import java.time.format.DateTimeFormatter;
 import java.time.temporal.ChronoUnit;
 
+
 public class Duke {
-    
+
     private static Storage storage;
     private TaskList inputs;
```

Comment:
Could the name "inputs" be a little more precise when it comes to referring a TaskList?